src/main/java/uk/gov/companieshouse/web/companydocuments/service/CompanyService.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.companieshouse.web.companydocuments.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.util.UriTemplate;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
+import uk.gov.companieshouse.web.companydocuments.api.ApiClientService;
+import uk.gov.companieshouse.web.companydocuments.exception.ServiceException;
+import uk.gov.companieshouse.web.companydocuments.model.CompanyDetail;
+import uk.gov.companieshouse.web.companydocuments.transformer.CompanyTransformer;
+
+@Service
+public class CompanyService {
+
+    @Autowired
+    private ApiClientService apiClientService;
+
+    @Autowired
+    private CompanyTransformer companyTransformer;
+
+    private static final UriTemplate COMPANY_URI = new UriTemplate("/company/{companyNumber}");
```

Comment:
Should be at top of file.

Reply:
Moved in [b916061](https://github.com/companieshouse/company-documents.web.ch.gov.uk/pull/10/commits/b9160619da9c40be5d6b3314aec0733f77b248a6) 👍